### ISSUE TITLE
Feature request: Save screen layout.

### ISSUE THREAD

**Issue by antrys** [TanninOne/modorganizer#796]
I'd be nice to save where the widths of each column and the windows are set to, instead of having to drag them around each session so I can see everything.

Thanks!

**Comment by TanninOne** [comment]
The screen layout including column width in mod and plugin list is being saved. Do you have a concrete example of something not being saved?

**Comment by antrys** [comment]
Yup.
![how i set it](https://cloud.githubusercontent.com/assets/13042832/15985296/8066b07c-2fb6-11e6-96ff-a48c2407e95e.png)
![how it is on next load](https://cloud.githubusercontent.com/assets/13042832/15985298/85427de2-2fb6-11e6-900b-3a50da4ee8a0.png)
![how i set it 2](https://cloud.githubusercontent.com/assets/13042832/15985302/961d4ac0-2fb6-11e6-9bde-10b660477a6c.png)
![how it is on next load 2](https://cloud.githubusercontent.com/assets/13042832/15985305/9b239196-2fb6-11e6-9569-40823824c357.png)

Thanks for responding! :-)

**Comment by filmcriticbeta** [comment]
From those screen shots it looks like you're using a high dpi monitor.  This might be the same issue I illustrated in #658 (QwindowsWindow::setgeometrydp  unable to set geometry) .

**Comment by antrys** [comment]
Yes, I am. I'm running a 4k display with 3x scaling.  I do, in fact have a lot of the issues you show in the other thread as well, but I've lived with them as I guess I've become accustomed to that behavior in some programs that haven't "caught up" with 4k displays.    The column with thing I've never associated with high DPI scaling before, hence the post.

**Comment by TanninOne** [comment]
Thanks for the screenshots.

**Comment by TanninOne** [comment]
Should be fixed in 2.0.7
